src/integrator_interface.jl

```diff
@@ -564,18 +580,27 @@ end
 
 last_step_failed(integrator::DEIntegrator) = false
 
+# Standard error message for classical PID type controllers
+function controller_message_on_dtmin_error(integrator::DEIntegrator)
+    if isdefined(integrator, :EEst)
+       return ", and step error estimate = $(integrator.EEst)"
+    else
+        return ""
+    end
+end
```

Comment:
Some voodoo time adaption algorithms actually work without defining an error estimate, so I would like to remove the dependency that the integrator itself needs to carry the error estimate and would like to propose in the future that there is either some kind of ControllerCache or the contoller itself carries the estimate.

Reply:
Yeah... that's going to be rough because there needs to be a tie-in in every stepper for how to calculate and store EEst, if it's needed. I'm not opposed though, because yes some methods like a priori estimates don't need the EEst and so it shouldn't spend the time to calculate it.